.github/actions/templates/avm-validateModuleDeployment/action.yml

```
@@ -211,7 +211,8 @@ runs:
           # Add custom parameters as needed
           if($moduleTemplatePossibleParameters -contains 'resourceLocation') {
             $functionInput.AdditionalParameters += @{
-              resourceLocation = '${{ steps.get-resource-location.outputs.resourceLocation }}'
+              # resourceLocation = '${{ steps.get-resource-location.outputs.resourceLocation }}'
+              resourceLocation = 'eastus'
```

```suggestion
              resourceLocation = '${{ steps.get-resource-location.outputs.resourceLocation }}'
```
Undo one done with testing